go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/lib/pq v1.10.9
 	github.com/moby/term v0.5.0
 	github.com/opencontainers/image-spec v1.1.0
-	github.com/opencontainers/runc v1.1.13
+	github.com/opencontainers/runc v1.1.14
```

Comment:
I'm not sure we need upgrading runc version in this PR. It's the job for @dependabot.

Reply:
iirc a check was failing, shall I revert?

Reply:
reverted

Reply:
@alexandear here is the failing check
https://github.com/ory/dockertest/actions/runs/11347631243/job/31559482556

Reply:
The maintainer merges #538 and the fail in this branch will be gone.